### ISSUE TITLE
Datapack loading

### ISSUE THREAD

**Issue by Illest8** [LangYueMc/EquipmentStandard#2]
Using the newly 1.1.5 release on the client-side, I've been directly editing the template datapack you provided with minecraft items to apply the attributes to and nothing's being applied in-game. I was wondering if you could post a light, minimalistic datapack that automatically works with vanilla minecraft items?

**Comment by Moirstral** [comment]
Default datapack not working in game?
I've tested it and it should be fine. There's no problem with LAN connection. It's theoretically possible. I'll write a custom data package tutorial when I have free time.

**Comment by Illest8** [comment]
Without modifying any files from the datapack you provided, I went ahead and added the item tag **equipment_standard:armor** onto diamond armor items. It doesn't show up in game.

I am very good at using datapacks so I understand the paths and logic to make sure they load correctly. I saw that the tag you provided with the built-in datapack is equipment_standard:armor, which is not shipped with the mod. Default datapack does not work in game.

**Comment by Illest8** [comment]
[equipment_standard_datapack.zip](https://github.com/LangYueMc/EquipmentStandard/files/13295581/equipment_standard_datapack.zip)

**Comment by Moirstral** [comment]
Is this your datapack?

If loaded into the game, it won't work because minecraft will not read it. The correct datapack format is like this:
```
(Datapack Name).zip
├ pack.mcmeta
└ data
   └ equipment_standard
      ├ template
         ├ temp.json
         └ ...
      ├ attr_score
         ├ score.json
         └ ...
```
[WIKI](https://minecraft.fandom.com/wiki/Data_pack)

**Comment by Illest8** [comment]
No that is not the exact path of my datapack. I have it set up the exact format you posted above.

**Comment by Illest8** [comment]
My datapack loads into the game but doesn't apply any changes from the folder I posted above inside of my datapack folder

**Comment by Moirstral** [comment]
You can try changing the file name first. The same name will be overwritten (Maybe).  I don’t have time to find out the reason these days.

**Comment by Moirstral** [comment]
https://github.com/LangYueMc/EquipmentStandard/blob/master/README_EN.md

**Comment by Illest8** [comment]
Thanks for making the guide. My datapack loads but still doesn't make any changes.

I made a bare bones datapack using your guide. Can you look at the 2 jsons and tell me what is wrong?

**Comment by Illest8** [comment]
File location: datapack/data/equipment_standard/attr_score/generic.crit_damage.json
[generic.crit_damage.json](https://github.com/LangYueMc/EquipmentStandard/files/13326771/generic.crit_damage.json)

File location: datapack/data/equipment_standard/template/ swords.json
[swords.json](https://github.com/LangYueMc/EquipmentStandard/files/13326772/swords.json)

**Comment by Moirstral** [comment]
![image](https://github.com/LangYueMc/EquipmentStandard/assets/19365612/a5cbcaa5-56ed-4d47-aef3-1f634d9f5756)
It is effective, but because this attribute is in the default datapack, the effect may not be obvious. You can find an attribute that is not in my default datapack, or use the same path file name as the default datapack to override the default.

**Comment by Moirstral** [comment]
[Test.zip](https://github.com/LangYueMc/EquipmentStandard/files/13327306/Test.zip)
Put it in `saves/Your world folder/datapacks` and then re-enter the world, I'm not sure if `/reload` will do anything.

**Comment by Illest8** [comment]
Went ahead and tested it. Still no effect. I ran the /datapack command and Test.zip loaded correctly so I don't know what the issue is. I only have the dependencies and the base mod downloaded.

**Comment by Illest8** [comment]
The attribute should show up on the tooltip right?

**Comment by Moirstral** [comment]
Items taken out in creative mode have no applied attributes, Need to be crafted, purchased from villagers, or looted.

**Comment by Illest8** [comment]
I tried crafting and using /give and nothing has changed

**Comment by Moirstral** [comment]
You can crafting a few more times, but /give confirms that it is invalid.  If you still have no additional attributes after crafting it multiple times, then there is nothing I can do. Maybe you can try [Apotheosis](https://www.curseforge.com/minecraft/mc-mods/apotheosis).

**Comment by Moirstral** [comment]
It has been fixed in dee179a69dadd32180d0c95190ab959538039206, and it has been tested and works. Sorry for the long wait.

**Comment by Illest8** [comment]
I'll give it a go! Thanks.
